Arrangement_on_surface_2/demo/Arrangement_on_surface_2/ArrangementPainterOstream.cpp

```diff
@@ -15,55 +15,48 @@
 
 #include <QGraphicsView>
 
-namespace CGAL
-{
-namespace Qt
-{
+namespace CGAL {
+namespace Qt {
 
-// Instantiation of Arr_segment_traits_2
+// Specialization of `ArrangementPainterOstream` for the segment traits.
 template <typename Kernel_>
 ArrangementPainterOstream<CGAL::Arr_segment_traits_2<Kernel_>>&
-ArrangementPainterOstream<CGAL::Arr_segment_traits_2<Kernel_>>::operator<<(
-  const X_monotone_curve_2& curve)
-{
+ArrangementPainterOstream<CGAL::Arr_segment_traits_2<Kernel_>>::
+operator<<(const X_monotone_curve_2& curve) {
   const Point_2& p1 = curve.source();
   const Point_2& p2 = curve.target();
   Segment_2 seg(p1, p2);
 
   // skip segments outside our view
   QRectF seg_bb = this->convert(seg.bbox());
-  if (
-    this->clippingRect.isValid() && !this->clippingRect.intersects(seg_bb) &&
-    (!seg.is_horizontal() && !seg.is_vertical()))
-  { return *this; }
+  if (this->clippingRect.isValid() &&
+      ! this->clippingRect.intersects(seg_bb) &&
+      ! seg.is_horizontal() && ! seg.is_vertical())
+    return *this;
 
   this->painterOstream << seg;
   return *this;
 }
 
-// Instantiation of Arr_polyline_traits_2
-
+// Specialization of `ArrangementPainterOstream` for the polyline traits.
 template <typename SegmentTraits>
 ArrangementPainterOstream<CGAL::Arr_polyline_traits_2<SegmentTraits>>&
 ArrangementPainterOstream<CGAL::Arr_polyline_traits_2<SegmentTraits>>::
-operator<<(const X_monotone_curve_2& curve)
-{
-  for (typename X_monotone_curve_2::Subcurve_const_iterator it =
-         curve.subcurves_begin();
-       it != curve.subcurves_end(); ++it)
-  {
+operator<<(const X_monotone_curve_2& curve) {
+  int cnt = 0;
```

Comment:
This triggers: `/home/cgal_tester/build/src/cmake/platforms/ArchLinux-clang-CXX20-Release/test/Arrangement_on_surface_2_Demo/ArrangementPainterOstream.cpp:46:7: warning: variable 'cnt' set but not used [-Wunused-but-set-variable]
  int cnt = 0;`